Congrats.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { StyleSheet, Text, View, Button, Image, Dimensions, TouchableOpacity } from 'react-native';
+import { Font } from 'expo';
+import PongButton from './PongButton';
+
+
+
+export class Congrats extends React.Component {
+  state = { fontLoaded: false };
+  async componentDidMount() {
+    await Font.loadAsync({
+      'double-bubble-shadow': require('./assets/fonts/Double_Bubble_shadow.otf'),
+      'source-sans-pro': require('./assets/fonts/SourceSansPro-Bold.ttf'),
+    });
+    this.setState({ fontLoaded: true });
+  }
+  render() {
+    return (
+      <View style={{ flexDirection: 'column' }}>
+        <View style={{ height: '58%', alignItems: 'center' }}>
+          {
+            this.state.fontLoaded ? (
+              <View style={{ height: '37%', alignItems: 'center' }}>
+                <Text style={ styles.congratsText }>It{ `'` }s a</Text>
+                <Text style={ styles.congratsText }>Match!</Text>
+              </View>
+            ) :
+            <View style={{ height: '37%', alignItems: 'center' }}>
+              <Text style={ styles.congratsTextBackup }>It{ `'` }s a</Text>
+              <Text style={ styles.congratsTextBackup }>Match!</Text>
+            </View>
+          }
+          <Image source={ require('./assets/image.png') }
+            style={{
+            tintColor: '#000000',
+            height: '40%',
+            width: '41%',
+            margin: 'auto',
+            marginBottom: 10,
+            }}
+          />
+          <Image source={ require('./assets/person.png') }
+            style={{
+              height: '15%',
+              width: '15%',
+              margin: 'auto',
+            }}
+          />
+        </View>
+        <View style={ styles.congratsContainer }>
+        {
+          this.state.fontLoaded ? (
+          <View style={ styles.congratsBox }>
+            <Text style={ styles.congratsInfo }>
+              Name: Jenny{ '\n' }Place: Phi Delt{ '\n' }Time: Now
+            </Text>
+          </View>
+          ) :
+          <View style={ styles.congratsBox }>
+            <Text style={ styles.congratsInfoBackup }>
+              Name: Jenny{ '\n' }Place: Phi Delt{ '\n' }Time: Now
+            </Text>
+          </View>
+        }
+        </View>
+        <TouchableOpacity onPress={ this._onPressButton } style={{ height: '6%' }}>
+          <View style={ styles.bottomButton }>
+          {
+            this.state.fontLoaded ? (
+              <Text style={ styles.buttonText }>tap to continue</Text>
+            ) :
+            <Text style={ styles.buttonTextBackup }>tap to continue</Text>
```

Comment:
There doesn't seem to be any styles defined for `buttonTextBackup`. Do you want to include new backup styles?